examples/lightning_base.py

```diff
@@ -245,7 +245,8 @@ def add_model_specific_args(parser, root_dir):
 
 class LoggingCallback(pl.Callback):
     def on_batch_end(self, trainer, pl_module):
-        lrs = {f"lr_group_{i}": param["lr"] for i, param in enumerate(pl_module.trainer.optimizers[0].param_groups)}
+        self.lr_scheduler = trainer.lr_schedulers[0]["scheduler"]
```

Comment:
Is `self.` necessary? A local variable seems to be good.

Reply:
I was just mimicking the rest of the code:
```
examples/seq2seq/finetune.py:        self.lr_scheduler = scheduler
examples/token-classification/run_pl_ner.py:        tensorboard_logs = {"loss": loss, "rate": self.lr_scheduler.get_last_lr()[-1]}
src/transformers/trainer.py:        self.optimizer, self.lr_scheduler = optimizers
src/transformers/trainer.py:        if self.lr_scheduler is None:
src/transformers/trainer.py:            self.lr_scheduler = get_linear_schedule_with_warmup(
src/transformers/trainer.py:            self.lr_scheduler.load_state_dict(torch.load(os.path.join(model_path, "scheduler.pt")))
src/transformers/trainer.py:                    self.lr_scheduler.step()
src/transformers/trainer.py:                            self.lr_scheduler.get_last_lr()[0]
src/transformers/trainer.py:                            else self.lr_scheduler.get_lr()[0]
src/transformers/trainer.py:                            xm.save(self.lr_scheduler.state_dict(), os.path.join(output_dir, "scheduler.pt"))
src/transformers/trainer.py:                            torch.save(self.lr_scheduler.state_dict(), os.path.join(output_dir, "scheduler.pt"))
src/transformers/trainer_tf.py:        self.optimizer, self.lr_scheduler = optimizers
src/transformers/trainer_tf.py:        if not self.optimizer and not self.lr_scheduler:
src/transformers/trainer_tf.py:            self.optimizer, self.lr_scheduler = create_optimizer(
src/transformers/trainer_tf.py:                        logs["learning_rate"] = self.lr_scheduler(self.global_step).numpy()
examples-orig/lightning_base.py:        lrs = {f"lr_group_{i}": lr for i, lr in enumerate(self.lr_scheduler.get_lr())}
examples-orig/seq2seq/finetune.py:        self.lr_scheduler = scheduler
examples-orig/token-classification/run_pl_ner.py:        tensorboard_logs = {"loss": loss, "rate": self.lr_scheduler.get_last_lr()[-1]}
examples-orig/text-classification/run_pl_glue.py:        tensorboard_logs = {"loss": loss, "rate": self.lr_scheduler.get_last_lr()[-1]}
```

Reply:
I think we should delete the `self.`